pkg/networkmanager/firewall.jsx

```diff
@@ -90,14 +97,26 @@ function ServiceRow(props) {
         deleteButton
     ];
 
-    var tabs = [];
+    let description, includes, simpleBody;
     if (props.service.description)
-        tabs.push({ name: _("Details"), renderer: () => <p>{props.service.description}</p> });
+        description = <p>{props.service.description}</p>;
+
+    if (props.service.includes.length > 0) {
+        includes = <React.Fragment>
+            <h5>Included Services</h5>
+            <ul>{props.service.includes.map(s => {
+                let service = firewall.services[s];
+                if (service && service.description)
+                    return <li key={service.id}><strong>{service.name}</strong>: {service.description}</li>;
+            })} </ul></React.Fragment>;
+    }
+    if (description || includes)
+        simpleBody = <React.Fragment>{description}{includes}</React.Fragment>;
```

Comment:
Would be a bit weird to have neither, but visually this shouldn't actually be different to what we have now.

Reply:
When neither are present the row is simply not expandable. This is the case in custom ports.